PYXAID2/runMD2.py

```diff
@@ -266,7 +266,7 @@ def compute_H_el(C_dia_a, C_dia_b, C_adi_a, C_adi_b, E_adi_a, E_adi_b):
 
 
 
-def compute_Hvib(coeff_curr, coeff_next, coeff_curr1, coeff_next1, E_adi_curr, E_adi_next,e_curr1, e_next1, params):
+def compute_Hvib(coeff_curr1, coeff_next1, coeff_curr0, coeff_next0, E_adi_curr1, E_adi_next1,e_curr0, e_next0, params):
```

Comment:
no - let's keep the order of variables in the argument list as is was - first - non-relativistic, second - relativistic. We should change the function body